### ISSUE TITLE
ESLint error: Unable to resolve path to module '@hookform/resolvers'

### ISSUE THREAD

**Issue by minimaluminium** [react-hook-form/resolvers#19]
Shows an ESLint error says `Unable to resolve path to module '@hookform/resolvers'`. I can use the resolver without any issue, just a lint error.

![Screen Shot 2020-07-07 at 2 10 43 PM](https://user-images.githubusercontent.com/1418797/86741324-de2fe000-c069-11ea-9a2a-58aa75283a6f.png)

**Comment by bluebill1049** [comment]
Thanks @minimaluminium can you try to remove node_module and install it again? I don't see the issue at my end. Mabye that will resolve the problem.

**Comment by minimaluminium** [comment]
Hi. Removed `node_modules` and installed again, but same as before. I imported like below and the error is gone.

```js
import {yupResolver} from '@hookform/resolvers/dist'
```

**Comment by bluebill1049** [comment]
hmmm this is strange.

**Comment by bluebill1049** [comment]
FYI @kotarella1110

**Comment by actual-size** [comment]
I'm getting a compile error that was also solved by that import:

`./node_modules/@hookform/resolvers/dist/index.esm.js
Attempted import error: 'appendErrors' is not exported from 'react-hook-form'.`

**Comment by bluebill1049** [comment]
thanks @actual-size which version of RHF you are using?

**Comment by kotarella1110** [comment]
@minimaluminium please provide `.eslintrc.js`.

**Comment by actual-size** [comment]
@bluebill1049 RHF `^5.7.2` and @hookform/resolvers `0.0.5`

**Comment by bluebill1049** [comment]
@actual-size v5 doesn't need resolver, resolver is for v6 :)

**Comment by minimaluminium** [comment]
@kotarella1110 Here it is.

```json
{
  "extends": ["wesbos"],
  "rules": {
    "no-nested-ternary": 0,
    "prettier/prettier": [
      "error",
      {
        "semi": false,
        "bracketSpacing": false,
        "singleQuote": true
      }
    ]
  }
}
```

**Comment by fcrezza** [comment]
use validationSchema for v5

`
const schema = // your schema

const {register, handleSubmit} = useForm({
  validationSchema: schema
})
`

**Comment by fcrezza** [comment]
use validationSchema for v5

`const schema = // your schema`

`const {register, handleSubmit} = useForm({`
`  validationSchema: schema`
`})`

**Comment by mrlubos** [comment]
Same issue and what @minimaluminium said fixed it, using react-hook-form 6.0.1

**Comment by bluebill1049** [comment]
@mrlubos is it eslint as well?

**Comment by snmgian** [comment]
https://github.com/react-hook-form/resolvers/pull/23

**Comment by bluebill1049** [comment]
Thanks, heaps @snmgian I will release a patch now.

**Comment by bluebill1049** [comment]
before I close this, how did you find out it was due to that? @snmgian

**Comment by snmgian** [comment]
My app's build process was running and ok, and the app was also running ok. Only eslint was complaining. When I faced those errors in the past, it was related to a package not being installed properly or not being installed at all. 

As package.json tells where's the file that is going to be imported, my first step was to look at the package.json. Then I found the missing `dist/` path in `jsnext:main`.

As `eslint-plugin-import` uses jsnext:main entry, I tested it locally by fixing that path, run the linter again with no errors this time.
